elements/lisk-p2p/src/p2p_types.ts

```diff
@@ -13,24 +13,40 @@
  *
  */
 import { ConnectionKind, PeerKind } from './constants';
-export interface P2PPacket {
-	readonly data?: unknown;
-}
 
-export interface P2PRequestPacket extends P2PPacket {
+export interface P2PRequestPacket {
 	readonly data?: unknown;
 	readonly procedure: string;
 }
 
-export interface P2PResponsePacket extends P2PPacket {
+export interface P2PResponsePacket {
```

Comment:
It will be nice to have it here too

Reply:
This interface is mainly used in P2PRequest class which intrinsically includes the peer ID.

Reply:
I meant it would be nice to have consistently returning the peerId on return action here
https://github.com/LiskHQ/lisk-sdk/blob/f5e2e3428957cd96eafc119dcba7a76d0bd99264/elements/lisk-p2p/src/p2p.ts#L610

Reply:
and probably https://github.com/LiskHQ/lisk-sdk/blob/f5e2e3428957cd96eafc119dcba7a76d0bd99264/elements/lisk-p2p/src/p2p.ts#L620
too (this one is kinda redundant)